docs/python_docs/python/tutorials/packages/autograd/index.md

```diff
@@ -39,7 +39,7 @@ Assuming we've calculated the gradient of each parameter with respect to the los
 
 We differentiate. [MXNet Gluon](/api/python/docs/tutorials/packages/gluon/index.html) uses Reverse Mode Automatic Differentiation (`autograd`) to backprogate gradients from the loss metric to the network parameters.
 
-![forward-backward](_static/autograd_forward_backward.png)
+![forward-backward](../../../../_static/autogluon_images/autograd_forward_backward.png)
```

Comment:
another option for these might be putting them here: https://github.com/dmlc/web-data/tree/master/mxnet/tutorials